wpilib/wpilib/samplerobot.py

```diff
@@ -28,17 +30,25 @@ class SampleRobot(RobotBase):
     Alternatively you can override the :meth:`robotMain` method and manage all
     aspects of the robot yourself (not recommended).
     
-    .. warning:: While it may look like a good choice to use for your code
-                 if you're inexperienced, don't. Unless you know what you
-                 are doing, complex code will be much more difficult under
-                 this system. Use :class:`.IterativeRobot` or command based
-                 instead if you're new.
+    .. deprecated:: 2018.0.0
+        While it may look like a good choice to use for your code
+        if you're inexperienced, don't. Unless you know what you
+        are doing, complex code will be much more difficult under
+        this system. Use :class:`.TimedRobot` or command based
+        instead if you're new.
     """
-    
+
     #: A python logging object that you can use to send messages to the log. It
     #: is recommended to use this instead of print statements.
     logger = logging.getLogger("robot")
 
+    def __init__(self):
+        warnings.warn("""While it may look like a good choice to use for your code if you're
+                            inexperienced, don't. Unless you know what you are doing, complex code will
+                            be much more difficult under this system. Use TimedRobot or Command-Based
+                            instead""", DeprecationWarning, stacklevel=2)
```

Comment:
This string still has newlines. That was not what I was suggesting.

Reply:
do you want it as one long string?